src/integrationTest/java/uk/gov/hmcts/darts/cases/service/RetentionPolicyTest.java

```diff
@@ -0,0 +1,172 @@
+package uk.gov.hmcts.darts.cases.service;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.hmcts.darts.cases.mapper.CasesMapper;
+import uk.gov.hmcts.darts.common.entity.CaseRetentionEntity;
+import uk.gov.hmcts.darts.common.entity.CourtCaseEntity;
+import uk.gov.hmcts.darts.common.entity.RetentionPolicyTypeEntity;
+import uk.gov.hmcts.darts.retention.enums.CaseStatus;
+import uk.gov.hmcts.darts.retention.enums.Policy;
+import uk.gov.hmcts.darts.testutils.IntegrationBase;
+
+import java.time.OffsetDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.darts.testutils.data.HearingTestData.createSomeMinimalHearing;
+
+@SuppressWarnings("VariableDeclarationUsageDistance")
+class RetentionPolicyTest extends IntegrationBase {
+
+    @Autowired
+    private CasesMapper casesMapper;
+
+    @Test
+    void singlePolicyWithCurrentStateComplete() {
+        var minimalHearing = createSomeMinimalHearing();
+        CaseRetentionEntity caseRetention = createCompleteCaseRetention(minimalHearing.getCourtCase());
+
+        List<CaseRetentionEntity> caseRetentionList = new ArrayList<>();
+        caseRetentionList.add(caseRetention);
+        minimalHearing.getCourtCase().setCaseRetentionEntities(caseRetentionList);
+
+        var hearingEntity = dartsDatabase.saveRetentionsForHearing(minimalHearing, caseRetentionList);
+        var singleCase = casesMapper.mapToSingleCase(hearingEntity.getCourtCase());
+
+        assertThat(singleCase.getRetentionPolicyApplied()).isEqualTo(caseRetention.getRetentionPolicyType().getPolicyName());
+        assertThat(singleCase.getRetainUntilDateTime()).isEqualTo(caseRetention.getRetainUntil());
+        assertThat(singleCase.getRetentionDateTimeApplied()).isEqualTo(caseRetention.getRetainUntilAppliedOn());
+    }
+
+    @Test
+    void twoPoliciesWithCurrentStateCompleteReturnLatest() {
+        var minimalHearing = createSomeMinimalHearing();
+        CaseRetentionEntity oldCaseRetentionCompleted = createCompleteCaseRetention(minimalHearing.getCourtCase());
+        CaseRetentionEntity latestCaseRetentionCompleted = createCompleteCaseRetention(minimalHearing.getCourtCase());
+        latestCaseRetentionCompleted.setRetainUntil(OffsetDateTime.parse("2020-08-31T15:20:30+08:00"));
```

Comment:
nothing technically wrong with it, but seems strange we are using +8 timezone.

Reply:
Will update, originally was used from provided examples